otter/test/utils.py

```diff
@@ -748,6 +749,24 @@ def nested_sequence(seq, get_effect=attrgetter('effect'),
         get_effect)
 
 
+def nested_parallel(parallel, fallback_dispatcher=base_dispatcher):
```

Comment:
nope, I forgot to pass it through :) I guess that means this deserves some unit tests... :P